src/util/astWalker.js

```diff
@@ -30,6 +30,19 @@ AstWalker.prototype.walk = function (ast, callback) {
   }
 }
 
+/**
+  * walk the given @astList
+  *
+  * @param {Object} astList  - ASTs of all the sources
+  * @param {Function} - callback used by AstWalker to compute response
+  */
+AstWalker.prototype.walkAstList = function (astList, callback) {
+  var walker = new AstWalker()
+  for (var k in astList) {
+    walker.walk(astList[k].AST, callback)
```

Comment:
When we get the attribute ```sources``` of the compilation result. we also get a list of sources (which contain *.AST). so I should rename the parameter sourceList .....